The job **`x86_64-gnu-llvm-16`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/8091394324/job/22110425142) [(plain)](https://github.com/rust-lang/rust/commit/9dfe5ef8b5496f5749223ef127d701563af4fe30/checks/22110425142/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=sisungo
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_f1f95229-03eb-4302-b82a-ea4593ffff80
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=9228522590f227e506b61ad4c9ec6d1b0216f4c5
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_f1f95229-03eb-4302-b82a-ea4593ffff80
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_f1f95229-03eb-4302-b82a-ea4593ffff80
GITHUB_TRIGGERING_ACTOR=sisungo
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/121777/merge
GITHUB_WORKFLOW_SHA=9228522590f227e506b61ad4c9ec6d1b0216f4c5
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
#12 writing image sha256:2cbec05d90581e513526915b33a06d13074ac7f55b4e9e209c1091a23692d5c0 done
#12 naming to docker.io/library/rust-ci done
#12 DONE 10.0s
##[endgroup]
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-16]
##[group]Clock drift check
  local time: Thu Feb 29 05:20:15 UTC 2024
  network time: Thu, 29 Feb 2024 05:20:15 GMT
  network time: Thu, 29 Feb 2024 05:20:15 GMT
##[endgroup]
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-16', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'change-id=99999999', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--set', 'build.optimized-compiler-builtins', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-16/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: change-id            := 99999999
---
##[endgroup]
Testing GCC stage1 (x86_64-unknown-linux-gnu -> x86_64-unknown-linux-gnu)
   Compiling y v0.1.0 (/checkout/compiler/rustc_codegen_gcc/build_system)
    Finished release [optimized] target(s) in 1.28s
     Running `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-codegen/x86_64-unknown-linux-gnu/release/y test --use-system-gcc --use-backend gcc --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools/cg_gcc --release --no-default-features --mini-tests --std-tests`
Using system GCC
Using system GCC
[BUILD] example
[AOT] mini_core_hello_world
/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools/cg_gcc/mini_core_hello_world
abc
---

---- [ui] tests/rustdoc-ui/lints/doc_cfg_hide.rs stdout ----
diff of stderr:

18 LL | #![doc(cfg_hide(doc))]
19    |  +
20 
- error: `#[doc(cfg_hide(...)]` takes a list of attributes
+ error: `#[doc(cfg_hide(...))]` takes a list of attributes
23    |
23    |
24 LL | #![doc(cfg_hide = "test")]
27    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
28    = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>
29 
29 
- error: `#[doc(cfg_hide(...)]` takes a list of attributes
+ error: `#[doc(cfg_hide(...))]` takes a list of attributes
32    |
32    |
33 LL | #![doc(cfg_hide)]

The actual stderr differed from the expected stderr.
Actual stderr saved to /checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-ui/lints/doc_cfg_hide/doc_cfg_hide.stderr
To update references, rerun the tests and pass the `--bless` flag
---
--- stderr -------------------------------
error: this attribute can only be applied at the crate level
##[error]  --> /checkout/tests/rustdoc-ui/lints/doc_cfg_hide.rs:9:7
   |
LL | #[doc(cfg_hide(doc))] //~ ERROR
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>
   = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>
   = note: read <https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#at-the-crate-level> for more information
  --> /checkout/tests/rustdoc-ui/lints/doc_cfg_hide.rs:2:9
   |
LL | #![deny(warnings)]
   |         ^^^^^^^^
   |         ^^^^^^^^
   = note: `#[deny(invalid_doc_attributes)]` implied by `#[deny(warnings)]`
help: to apply to the crate, use an inner attribute
   |
LL | #![doc(cfg_hide(doc))] //~ ERROR
   |  +

error: `#[doc(cfg_hide(...))]` takes a list of attributes
   |
   |
LL | #![doc(cfg_hide = "test")] //~ ERROR
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>


error: `#[doc(cfg_hide(...))]` takes a list of attributes
   |
   |
LL | #![doc(cfg_hide)] //~ ERROR
   |
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>

```

</details>

--- a/tests/rustdoc-ui/lints/doc_cfg_hide.stderr
+++ b/tests/rustdoc-ui/lints/doc_cfg_hide.stderr
@@ -18,7 +18,7 @@ help: to apply to the crate, use an inner attribute
 LL | #![doc(cfg_hide(doc))]
    |  +
 
-error: `#[doc(cfg_hide(...)]` takes a list of attributes
+error: `#[doc(cfg_hide(...))]` takes a list of attributes
   --> $DIR/doc_cfg_hide.rs:4:8
    |
 LL | #![doc(cfg_hide = "test")]
@@ -27,7 +27,7 @@ LL | #![doc(cfg_hide = "test")]
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #82730 <https://github.com/rust-lang/rust/issues/82730>
 
-error: `#[doc(cfg_hide(...)]` takes a list of attributes
+error: `#[doc(cfg_hide(...))]` takes a list of attributes
   --> $DIR/doc_cfg_hide.rs:6:8
    |
 LL | #![doc(cfg_hide)]

